.storybook/decorators.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { action } from '@storybook/addon-actions';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+const storyMiddleware = (store: any) => (next: (act: any) => any) => (
+  act: any,
+) => {
+  action('dispatch');
+  return next(act);
+};
+
+export const withRedux = (state = {}) => (Story: React.ComponentClass) => {
+  const store = configureStore([thunk, storyMiddleware])(state);
+  return (
+    <Provider store={store}>
+      <Story />
+    </Provider>
+  );
+};
```

Comment:
@jgerigmeyer  I think I get why we need to wrap some components with this decorator (ie the TaskTable that gets data from the store). Wondering if when this is added, we don't need the IconSetting wrapper?

Reply:
The decorators returned from `preview.js` are global decorators (https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) that are always used for all stories. This file is for custom decorators that we can use or not use as we like -- but even when used, they are _in addition to_ the global decorators, not replacing.

(`TaskTable` doesn't actually use the store, but it does have the `AssignUsersModal` embedded, and that child component accesses the `user` object from the store.)